include/Photo.php

```diff
@@ -1059,3 +1065,23 @@ function store_photo($a, $uid, $imagedata = "", $url = "") {
 	return($image);
 }
 
+function photo_albums($uid, $update = false) {
+	$sql_extra = permissions_sql($uid);
+
+	$key = "photo_albums:".$uid.":".local_user().":".remote_user();
+	$albums = Cache::get($key);
+	if (is_null($albums) OR $update) {
+		/// @todo This query needs to be renewed. It is really slow
+		// At this time we just store the data in the cache
+		$albums = qu("SELECT count(distinct `resource-id`) AS `total`, `album`
```

Comment:
I  think `count` and `distinct` should be uppercase because they are sql commands